vkrunner/vr-config.c

```diff
@@ -27,15 +27,30 @@
 
 #include "vr-config-private.h"
 
+#include <assert.h>
 #include <string.h>
 
 void
-vr_config_add_script(struct vr_config *config,
-                     const char *filename)
+vr_config_add_script_file(struct vr_config *config,
+                          const char *filename)
 {
-        struct vr_config_script *script =
-                vr_alloc(sizeof *script + strlen(filename) + 1);
-        strcpy(script->filename, filename);
+        struct vr_config_script *script;
+        assert(filename && "|filename| must be a valid pointer");
```

Comment:
I think these asserts are not really necessary because the program would crash very quickly anyway if the assert failed. I would prefer not to have them to avoid littering the code.

Reply:
Removed.